examples/repl-derive.rs

```diff
@@ -0,0 +1,79 @@
+use std::io::Write;
+
+use clap::{Args, Parser, Subcommand};
+
+#[derive(Debug, Parser)]
+#[command(multicall = true)]
+struct Cli {
+    #[command(subcommand)]
+    command: Commands,
+}
+
+#[derive(Debug, Subcommand)]
+enum Commands {
+    Echo(EchoArgs),
+    Ping,
+    Exit,
+}
+
+#[derive(Args, Debug)]
+pub struct EchoArgs {
+    #[arg(
+        short = 't',
+        long = "text",
+        visible_alias = "text",
+        help = "The text to be echoed",
+        help_heading = "Echo"
+    )]
+    text: String,
+}
```

Comment:
Could you shrink this to match `repl.rs`s CLI.

Derive and builder examples should not deviate